--- grilo-plugins-0.1.17/configure.ac.wiget	2011-10-07 18:29:26.017818738 +0200
+++ grilo-plugins-0.1.17/configure.ac	2011-10-07 18:29:36.447488042 +0200
@@ -130,7 +130,7 @@ PKG_CHECK_MODULES(GMIME, gmime-2.6,
                                                    HAVE_GMIME=yes,
                                                    HAVE_GMIME=no)])
 
-PKG_CHECK_MODULES(TRACKER_SPARQL, tracker-sparql-0.11,
+PKG_CHECK_MODULES(TRACKER_SPARQL, tracker-sparql-0.12,
                                   HAVE_TRACKER_SPARQL=yes,
                                   [PKG_CHECK_MODULES(TRACKER_SPARQL, tracker-sparql-0.10 >= 0.10.5,
                                                                      HAVE_TRACKER_SPARQL=yes,
